bin_git/run_flake8.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
```

Comment:
For me, that's not directly git related - although you used it as git hooks.
It's about good coding styles and scripts that help one ensure this.
In addition, this can also be used by git and the CI system.
Therefore, I'd just put this directly into the ./utils folder. I also added a `run_pytest_cov.sh` script.

Reply:
I'll add information for run_flake8.sh and run_pytest_cov.sh to the documentation `working_practises`